tests/inputs/qchem/test_input_qchem_blocks.py

```
@@ -0,0 +1,268 @@
+    expected_dict = {
+        "METHOD": "hf",
+        "BASIS": "sto-3g",
+        "JOBTYPE": "sp",
+        "UNRESTRICTED": "False",
+        "SYMMETRY": "False",
+        "SYM_IGNORE": "True",
+    }
+    actual_block_full_string = inp._get_rem_block()
+    actual_block_content_lines = actual_block_full_string.strip().split("\n")
+    if len(actual_block_content_lines) < 2:
+        assert False, f"REM block is too short: {actual_block_full_string}"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)